Sources/GRPC/Server.swift

```diff
@@ -384,6 +390,24 @@ extension Server {
     /// the need to recalculate this dictionary each time we receive an rpc.
     internal var serviceProvidersByName: [Substring: CallHandlerProvider]
 
+    #if canImport(Network)
+    @available(macOS 10.14, iOS 12.0, watchOS 6.0, tvOS 12.0, *)
+    public var serverBootstrapNWParametersConfigurator: (
+      @Sendable (NIOTSListenerBootstrap) -> Void
+    )? {
```

Comment:
Similar feedback here, we can drop `serverBootstrap` and the closure should take a `NWParameters`